apps/docs/docs/reference/ui/organisms/Hero.mdx

```diff
@@ -0,0 +1,163 @@
+
+```tsx
+import { Hero, HeroImage, HeroHeading } from '@faststore/ui'
+```
+
+## Usage
+
+<Tabs>
+  <TabItem value="react" label="React" default>
+
+```tsx live
+  <Hero>
+    <HeroImage>
+      <img
+        alt="Controller on a table"
+        src="https://storeframework.vtexassets.com/arquivos/ids/190897/Photo.jpg"
+      />
+    </HeroImage>
+    <HeroHeading>
+      <h1>New Products Available</h1>
+      <p>
+        At BaseStore you can shop the best tech of 2022. Enjoy and get 10% off
+        on your first purchase.
+      </p>
+      <a href="/">
+        See all <Icon component={<RightArrow />} />
+      </a>
+    </HeroHeading>
+  </Hero>
+```
+
+  </TabItem>
+  <TabItem value="css" label="CSS">
+
+```css
+[data-store-hero] {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  min-height: 20rem;
+  background-color: #001947;
+  color: #fff;
+}
+
+[data-hero-heading] {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 2rem;
+}
+
+[data-hero-heading] h1 {
+  font-size: 1.81rem;
+  font-weight: bold;
+  margin-bottom: 1.5rem;
+}
+
+[data-hero-heading] a {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+  margin-top: 2rem;
+  min-width: 11.25rem;
+  width: fit-content;
+  color: #001947;
+  border: 1px solid #171a1c;
+  background-color: #fff;
+}
+
+[data-hero-image] {
+  height: 100%;
+  overflow: hidden;
+}
+
+@media only screen and (min-width: 769px) {
+  [data-store-hero] {
+    flex-direction: row-reverse;
+  }
+  [data-hero-heading] {
+    padding: 0 4rem;
+  }
+  [data-hero-heading] h1 {
+    font-size: 3rem;
+  }
+}
+```  
+
+  </TabItem>
+</Tabs>
+
+### Use cases
+
+Use the `Hero` component as the first element of your Home, brand, or collections pages. 
+
+## Props
+
+All hero-related components support all attributes also supported by the `<div>` tag. Besides those attributes, the following props are also supported:
+
+### Hero
+
+<PropsSection name="Hero" />
+
+### HeroImage
+
+<PropsSection name="HeroImage" />
+
+### HeroHeading
+
+<PropsSection name="HeroHeading" />
+
+## Customization
+
+`data-store-hero`
+
+`data-store-hero-image`
+
+`data-store-hero-heading`
+
+## Best practices
+
+### Do's
+
+- Draw a clear connection between the Hero image and text.
+- Use strong contrasts to make call-to-action buttons stand out.
+- Ensure the copy is legible on the top of the imagery.
+- Use optimized images for your Hero image to avoid harming your website performance. Notice that if the Hero banners take too long to load, they may lose efficacy.
+- Use an eye-catching image that adds value to your page. Hero images have a significant impact on your brand perception, website traffic, and sales conversion rate.
+- Follow your brand identity and guidelines. Remember that this is the first touchpoint shoppers will have with your brand.
+- Keep your message clear and connected with your visuals.
+- Make your Hero component responsive.
```

Comment:
```suggestion
#### Content
- Draw a clear connection between the Hero image and text.
- Ensure the copy is legible on the top of the imagery.
- Keep your message clear and connected with your visuals.

#### Visual
- Use strong contrasts to make call-to-action buttons stand out.
- Follow your brand identity and guidelines. Remember that this is the first touchpoint shoppers will have with your brand.
- Make your Hero component responsive.

#### Image
- Use optimized images for your Hero image to avoid harming your website performance. Notice that if the Hero banners take too long to load, they may lose efficacy.
- Use an eye-catching image that adds value to your page. Hero images have a significant impact on your brand perception, website traffic, and sales conversion rate.
```